sql/parser/rename.go

```diff
@@ -34,5 +34,5 @@ type RenameTable struct {
 }
 
 func (node *RenameTable) String() string {
-	return fmt.Sprintf("RENAME TABLE %s %s", node.Name, node.NewName)
+	return fmt.Sprintf("RENAME TABLE %s %s", encIdent(node.Name), encIdent(node.NewName))
```

Comment:
done. I refrained from messing with the grammar (b/c no idea about the consequences) and so we now have

```go
type Name string
type NameList []string // (not []Name)
```